src/constant/error.ts

```diff
@@ -1,14 +1,14 @@
 export const ERRORS = {
-  APP_FOLDER_EXISTED: 'App folder existed',
-  DOMAIN_STARTS_WITH_HTTPS: 'Domain has to start with https',
-  MISSING_APP_NAME: 'Missing app name',
-  PACKAGE_JSON_NOT_FOUND: 'package.json not found',
+  APP_FOLDER_EXISTED: 'App folder existed.',
+  DOMAIN_STARTS_WITH_HTTPS: 'Domain has to start with https.',
+  MISSING_APP_NAME: 'Missing app name.',
+  PACKAGE_JSON_NOT_FOUND: 'package.json not found.',
   PROJECT_FOLDER_EXISTS:
-    'Project folder already exists! Please, run the cli again and choose another project name.',
-  VALID_DOMAIN: 'Please enter a valid domain',
-  APP_NAME_MISSING: 'App name missing',
+    'Project folder already exists! Please run the cli again and choose another project name.',
+  VALID_DOMAIN: 'Please enter a valid domain.',
+  APP_NAME_MISSING: 'App name missing.',
   USER_NAME_EMPTY: "Username can't be empty.",
-  PASSWORD_EMPTY: "Password can't be empty",
+  PASSWORD_EMPTY: "Password can't be empty.",
   APP_ID_EMPTY: "App ID can't be empty.",
```

Comment:
Please use a single quotation if it's ok.

Reply:
In code
- We can't use this format: 'Password can't be empty.'
- We can use this format: 'Password can\'t be empty.', but prettier is not recommended.
- So I use "Password can't be empty."

Reply:
I finally understood. It's due to the single quotation in "can't".
How about using "cannot" instead?

Reply:
Sorry for the wrong description!!, I missing '\' in description above:

We can't use this format: 'Password can't be empty.'
We can use this format: 'Password can\'t be empty.', but prettier is not recommended.
So I use "Password can't be empty."

I updated code in here:
https://github.com/kintone-labs/kintone-cli/pull/118/commits/c47891cc978ff307674cac64d74f9357b6b4b769

Reply:
I see!
Thanks for the modification. I confirmed.